src/js/utils/language.js

```diff
@@ -100,6 +100,10 @@ export function isRtl(message) {
     return message.charCodeAt(0) === 8207 || rtlRegex.test(message);
 }
 
+export function isLanguageRtl(language) {
+    return ['ar', 'arc', 'dv', 'fa', 'ha', 'he', 'khw', 'ks', 'ku', 'ps', 'ur', 'yi'].indexOf(normalizeLanguageCode(language)) >= 0;
+}
```

Comment:
I don't like having to add this utility for a single field. If we're able to achieve automatic RTL layout on other aspects of the UI, we should be able to do the same for this element in the controlbar, otherwise we'd need this workaround for the entire player UI.